storage/innobase/xtrabackup/doc/source/installation/apt_repo.rst

```diff
@@ -108,6 +108,46 @@ Now you can install *Percona XtraBackup* by running:
   Installing the packages manually like this, you must
   resolve all dependencies and install the missing packages yourself.
 
+Update the Curl utility in Debian 10
+=============================================
+
+The default curl version, 7.64.0, in Debian 10 has known issues when attempting to reuse an already closed connection. This issue directly affects ``xbcloud`` and users may see intermittent backup failures. 
+
+For more details, see `curl #3750 <https://github.com/curl/curl/issues/3750>`__ or `curl #3763 <https://github.com/curl/curl/pull/3763>`__. 
+
+Follow these steps to upgrade curl to version 7.74.0: 
+
+#. Remove the current version of curl:
+
+   .. code-block:: bash
+
+      $ sudo apt remove curl
```

Comment:
I think that by doing this, we will also remove other software that has curl as a dependency

Reply:
Tested and verified that I can drop this step and curl is updated without issues.